### PR TITLE
[GCI-263] Rename variables names to comply with naming conventions

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/GCI-263

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java [hunk 1]
```diff
@@ -764,8 +764,8 @@ public GlobalProperty getGlobalPropertyByUuid(String uuid) {
 	 * @return memory in mega bytes
 	 */
 	private String convertToMegaBytes(long bytes) {
-		int ONE_KILO_BYTE = 1024;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java]: Did you take a look at our convention for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Comment (BartlomiejRasztabiga) [reply api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java]: Yes, but this variable wasn't made final by the author.

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java]: does it change?

Comment (BartlomiejRasztabiga) [reply api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java]: Fixed :)

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa This is ready to merge 😊

**Comment by dkayiwa** [comment]
@BartlomiejRasztabiga the ticket status does not indicate that it is ready for review. 😊

**Comment by BartlomiejRasztabiga** [comment]
Sorry, fixed :)

**Pull request merged**